independent-publisher-connectors/AfterShip/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2690 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "AfterShip",
+    "description": "AfterShip - Shipment tracking API"
+  },
```

Comment:
@taiki-yoshida The metadata is missing for your connector. Please make sure that you are following all the instructions mentioned in the [documentation](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#step-3-add-metadata).

Docs - [Add metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#step-3-add-metadata)

Reply:
@taiki-yoshida Please resolve this comment to get your PR reviewed again.

Reply:
@taiki-yoshida I can not see your changes. Can you please let me know where did you add the required metadata ?